test/python/rest/TestApplication/Main.spl

```diff
@@ -0,0 +1,71 @@
+/*
+** Copyright (C) 2017  International Business Machines Corporation
+** All Rights Reserved
```

Comment:
Not the correct copyright - since this is ALv2 licenced code. See existing files for standard copyright.

Reply:
No problem.